Applications/spire/source/book_view/quote_panel.cpp

```diff
@@ -25,14 +25,28 @@ QuotePanel::QuotePanel(const BookViewModel& model, Side side,
   label_layout->setSpacing(0);
   label_layout->addStretch(1);
   m_price_label = new QLabel(this);
+  m_price_label->setFixedWidth(scale_width(52));
   m_price_label->setAlignment(Qt::AlignCenter);
   m_price_label->setStyleSheet(QString(R"(
     color: #4B23A0;
     font-family: Roboto;
     font-size: %1px;
     font-weight: 550;)").arg(scale_height(12)));
   label_layout->addWidget(m_price_label);
+  auto separator = new QLabel("/", this);
+  separator->setAlignment(Qt::AlignRight | Qt::AlignCenter);
+  separator->setFixedWidth(scale_width(9));
+  separator->setIndent(0);
+  separator->setStyleSheet(QString(R"(
+    color: #4B23A0;
+    font-family: Roboto;
+    font-size: %1px;
+    padding-top: %2px;
+    font-weight: 550;)").arg(scale_height(10)).arg(scale_height(1)));
+  label_layout->addWidget(separator);
   m_size_label = new QLabel(this);
+  m_size_label->setFixedWidth(scale_width(52));
+  m_size_label->setIndent(scale_width(3));
```

Comment:
This indentation looks suspicious too.

Ultimately the layout engine should be used to ensure that the labels are placed correctly. The price and size labels should expand to take up as much available space as they can with equal proportion and the separator label should use up the minimum amount of width requested by the label.